server/controller/index.js

```diff
@@ -1,11 +1,82 @@
+const moment = require('moment');
 const model = require('../model');
 
 module.exports = {
   get: (req, res) => {
+    const calendarMonths = [];
+    const dateObject = moment();
+    for (let i = 0; i < 24; i += 1) {
+      const firstDayOfMonth = () => {
+        const firstDay = moment(dateObject).startOf('month').format('d');
+        return firstDay;
+      };
+
+      const blanks = [];
+      for (let j = 0; j < firstDayOfMonth(); j += 1) {
+        blanks.push(null);
+      }
+
+      const days = [];
+      for (let k = 1; k <= dateObject.daysInMonth(); k += 1) {
+        days.push(k);
+      }
+
+      calendarMonths.push({
+        month: dateObject.format('MMMM'),
+        year: dateObject.year(),
+        days: [...blanks, ...days.map((day) => ({ day, avail: true }))],
+      });
+      dateObject.add(1, 'M');
+    }
+
     const { propertyId } = req.params;
     model.Booking.find({ propertyId })
       .then((data) => {
-        res.send(data);
+        data.forEach(({ date }) => {
+          const dateStart = moment(date.start, 'MM-DD-YYYY');
+          const dateEnd = moment(date.end, 'MM-DD-YYYY');
+
+          const dayStart = +dateStart.format('D');
+          const monthStart = dateStart.format('MMMM');
+          const monthEnd = dateEnd.format('MMMM');
+          const yearStart = dateStart.year();
+          const yearEnd = dateEnd.year();
+
```

Comment:
This seems like a lot of logic to have on your index.js for your controller. I would recommend that you separate it into a separate file and call on it here, just to deconflict confusion on server routes and controls